worlds/lufia2ac/docs/setup_en.md

```diff
@@ -0,0 +1,159 @@
+# Lufia II Ancient Cave Multiworld Setup Guide
+
+## Required Software
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). Make sure to check the box for `SNI Client - Lufia II Ancient Cave Patch Setup`
+- Hardware or software capable of loading and playing SNES ROM files
+    - An emulator capable of connecting to SNI
+      ([snes9x rr](https://github.com/gocha/snes9x-rr/releases),
+       [BizHawk](http://tasvideos.org/BizHawk.html), or
+       [RetroArch](https://retroarch.com?page=platforms) 1.10.1 or newer). Or,
```

Comment:
Below specifies 1.10.3 or newer. Requiring either is fine IMO, just same version in both places in the file.
- 1.10.1 has no known bugs for LoROM games,
- on the other hand, someone implementing a HiROM game might just copy this guide. Some HiROM games need a bugfix in RA 1.10.3.

Reply:
adressed